pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -39,21 +39,26 @@ type Reconciler struct {
 	securitycli   securityclient.Interface
 
 	restConfig *rest.Config
-	verFixed   *version.Version
+	verFixed46 *version.Version
+	verFixed47 *version.Version
 }
 
+var (
+	verFixed47, _ = version.ParseVersion("4.7.18")  // fixed in 4.7.18
+	verFixed46, _ = version.ParseVersion("4.6.38 ") // fixed in 4.6.38
```

Comment:
it was actually fixed in 4.6.37: https://github.com/openshift/cluster-network-operator/blob/29129f31fd010b2596e62fabffe7dbc1896a81e8/bindata/network/openshift-sdn/sdn.yaml#L235

Reply:
Opened https://github.com/Azure/ARO-RP/pull/1621